python/python/bystro/ancestry/listener.py

```diff
@@ -25,33 +25,6 @@
 
 ANCESTRY_TUBE = "ancestry"
 
-class AncestryJobData(BaseMessage, frozen=True, rename="camel"):
```

Comment:
I had to move this, because listener.py can be a __main__ module. See https://github.com/jcrist/msgspec/issues/394